lib/errors/csv_dob_validation_error.rb

```diff
@@ -0,0 +1,2 @@
+class CsvDobValidationError < StandardError
```

Comment:
how about to move this class in the `app/services/additional_claimants_csv/validator.rb` and spare the new file and the extra path autoload?